### PR TITLE
Header section improvements 

### PR DESCRIPTION
**Pull request by DavDan1**
PT: https://www.pivotaltracker.com/story/show/178585249
____
### Description
Fix bugs in hero section, make styling improvement according to Jens suggestions.
____
### Changes proposed
- More opacity on wizard
- Less info in main view for phone
- Make smaller space between the header lines
- More margin from sides to content in wizard
- More margin in question bubble
- Add a dash before the answer text (example - we are 5)
- Bigger/thicker font weight in number inputs 
- Make input field feel more alive
- Change Jens face to another person
- check Swedish
- fix dropdown to open up

### CODE REVIEW

File: src/styles/_questionnaire.scss [hunk 1]
```diff
@@ -110,6 +113,10 @@
       margin-bottom: 10px;
     }
 
+    &[type='number'] {
+      font-size: 1.7rem;
+    }
+
```
Comment (LucasKnudsen) [inline src/styles/_questionnaire.scss:119]: Does this effect phone number?

File: src/styles/_questionnaire.scss [hunk 2]
```diff
@@ -239,6 +267,15 @@
       width: 300px;
       margin-right: 10px;
       color: $secondaryColor;
+
+      .rrs__button + .rrs__options {
+        top: auto;
+        bottom: 100% ;
+        @media only screen and (max-width: $mobile) {
+          top: 60px;
+          bottom: auto;
```
Comment (LucasKnudsen) [inline src/styles/_questionnaire.scss:276]: Pretty sm@rt

File: src/styles/_questionnaire.scss [hunk 3]
```diff
@@ -133,6 +140,19 @@
       @extend .input;
       background-color: $secondaryColor;
       color: $fontColor;
+      &:hover {
+        background-color: $secondaryColor;
+        transform: none;
+        transition: none;
```
Comment (LucasKnudsen) [inline src/styles/_questionnaire.scss:146]: Whai?

**Review by LucasKnudsen — APPROVED** [review]

File: src/styles/_questionnaire.scss (hunk 3 above)
Comment (offcast-dev) [reply src/styles/_questionnaire.scss:146]: When disabled we dont want inputs to animate, this gives user understanding that you cannot change it without clicking edit

File: src/styles/_questionnaire.scss (hunk 1 above)
Comment (arunbhalli) [reply src/styles/_questionnaire.scss:119]: it does effect phone number.

File: src/styles/_questionnaire.scss (hunk 3 above)
Comment (LucasKnudsen) [reply src/styles/_questionnaire.scss:146]: KUL

**Review by lukeperrin10 — APPROVED** [review]

**Pull request merged**